entrypoint.sh

```diff
@@ -17,6 +17,11 @@ then
     done
 fi
 
+if [[ ! -z "${CONTEXT}" ]]; then
```

Comment:
Idea: what about instead of an env var we do this through an [input](https://github.com/koslib/helm-eks-action/blob/master/action.yml#L6)? My logic here is that this will make it easier to understand, read and maintain.

Reply:
It seems you've implemented the main bullets of this change (which were the cli updates) by yourself, so I am closing this PR

Reply:
It was implemented by #29 :) Feel free to submit another PR for the context input though!